sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/application-restriction-bypass.aest.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Param, ParseIntPipe } from "@nestjs/common";
+import BaseController from "../BaseController";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { ApplicationRestrictionBypassSummaryAPIOutDTO } from "./models/application-restriction-bypass.dto";
+import { ClientTypeBaseRoute } from "../../types";
+import { ApiTags } from "@nestjs/swagger";
+import { ApplicationRestrictionBypassService } from "../../services";
+
+/**
+ * Controller for AEST Application Restriction Bypasses.
+ * This consists of all Rest APIs for AEST application restriction bypasses.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("application-restriction-bypass")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-application-restriction-bypass`)
+export class ApplicationRestrictionBypassAESTController extends BaseController {
+  constructor(
+    private readonly applicationRestrictionBypassService: ApplicationRestrictionBypassService,
+  ) {
+    super();
+  }
+
+  /**
+   * Get restriction bypasses for a given application.
+   * @param applicationId id of the application to retrieve restriction bypasses.
+   * @returns Application restriction bypasses.
```

Comment:
Suggestion: To use small case for `application`.

Reply:
Updated.